src/components/WalletDropdown/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useWeb3React } from '@web3-react/core'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+import DefaultMenu from './DefaultMenu'
+import { LanguageMenu } from './LanguageMenu'
+import { TransactionHistoryMenu } from './TransactionMenu'
+
+const WalletWrapper = styled.div<{ isAuthenticated: boolean }>`
+  border-radius: 12px;
+  width: 320px;
+  height: ${({ isAuthenticated }) => (isAuthenticated ? '324px' : '200px')};
```

Comment:
I think we want to set the height of everything except the Authenticated section as 200. then the authenticated header has a height of 124px. this way you won't need a conditional here

Reply:
I had to change this slightly just now to account for when the claim button renders.  I think what I have here is the simplest given the requirements that the wallet always stays the same height when switching between the various menus.  Otherwise i have to keep track of height in multiple components